cli/src/main/java/com/devonfw/tools/ide/tool/ide/IdeaBasedIdeToolCommandlet.java

```diff
@@ -38,7 +38,6 @@ public IdeaBasedIdeToolCommandlet(IdeContext context, String tool, Set<Tag> tags
   protected void configureToolArgs(ProcessContext pc, ProcessMode processMode, ProcessErrorHandling errorHandling, String... args) {
 
     super.configureToolArgs(pc, processMode, errorHandling, args);
-    pc.addArg(this.context.getWorkspacePath());
```

Comment:
I removed that part and I am executing `run()` after the plugin installation is complete (in case that is the idea of that part). I am not sure if that change breaks anything. Otherwise, I need some more clarification on why the workspace path is added to the arguments and where I could move it

Reply:
An alternative could be to check if the `args` start with `installPlugins` but it is also OK to remove it here and move the argument to `runIde`. But then you should remove the enitre method since just calling `super` is kind of pointless now.